@here/harp-theme-tool/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@here/harp-theme-tools",
```

Comment:
Is there an easy way to test this. If I do npx harp-theme-optimizer it tries to download it from the npm registry.

Reply:
I renamed the folder to `harp-theme-tools`. Makes sense b/c there will be more tools in the future and it is also consistent with the `harp-atlas-tools`

Reply:
> Is there an easy way to test this. If I do npx harp-theme-optimizer it tries to download it from the npm registry.

I don't know, but for sure.

1) You need shebabg in `src/cli-build-theme.ts`
2) `yarn prepare && npx -p . harp-theme-optimizer` works more or less but not perfect, because `npx` installs all deps in local `node_modules` ignoring fact that we're already in `yarn workspace` and susequent `yarn prepare` fails  because it cannot resolve new unpublished APIs:

```
$ yarn prepare && npx --no-install -p . harp-theme-optimizer
yarn run v1.19.2
$ cross-env tsc --sourceMap false $EXTRA_TSC_ARGS
src/cli-build-theme.ts:47:9 - error TS2345: Argument of type '{ resolveResourceUris: boolean; resolveIncludeUris: boolean; }' is not assignable to parameter of type 'ThemeLoadOptions'.
  Object literal may only specify known properties, and 'resolveResourceUris' does not exist in type 'ThemeLoadOptions'.

47         resolveResourceUris: false,
```

So, unfortunately i have no clear guideline how to proceed with testing.

> I renamed the folder to harp-theme-tools
+1

Reply:
I tried this:
```
yarn prepare && npx --no-install -p . harp-theme-optimizer
```
but the behavior is really weird. It seems to pull the latest harp-mapview from npm instead of taking it from source. Even more weird: After running yarn prepare, yarn postinstall in mapsdk is also not working anymore even though I run the ts version of the script explicitly.